react-native/react/images/folders/fa-folder-private-empty.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
i'm fine if they're copies, but let's not name them fa- cause they're not font-awesome things and that's confusing